bin/cron.js

```diff
@@ -88,6 +334,9 @@ cron.schedule("*/15 * * * *", async () => {
       });
       badges.value.map((badge) => {
         badgeIdToPoints[badge._id] = parseInt(badge.Points);
+        if (badge._id === 32 && typeof badge._id.assignment_id !== "undefined") {
```

Comment:
`badge._id.assignment_id` ?? Isn't badge._id a number?

Reply:
Yea you're right will change it